orthw

```diff
@@ -237,6 +237,8 @@ create_package_configuration() {
       --package-id $package_id \
       --create-hierarchical-dirs \
       --generate-path-excludes \
+      --license-classifications-file $ort_config_license_classifications_file \
+      --non-offending-license-categories "$non_offending_license_categories" \
```

Comment:
I am not sure I would like the default behavior of `create_package_configuration()` to be only `non-offending-license-categories`.

Reply:
Updated the description of the corresponding configuration property. I believe the behavior is / was like you want it to be.